easyconfigs/m/modbam2bed/modbampy-htslib-deflate.patch

```diff
@@ -0,0 +1,101 @@
+diff -ruN ../modbam2bed.orig/build.py ../modbam2bed/build.py
+--- ../modbam2bed.orig/build.py	2024-10-30 13:56:33.569032588 +0000
++++ ../modbam2bed/build.py	2024-10-30 19:05:08.390020330 +0000
+@@ -5,13 +5,13 @@
+ 
+ dir_path = os.path.dirname(os.path.realpath(__file__))
+ src_dir='src'
+-libraries=['m', 'lzma', 'bz2', 'pthread', 'curl', 'crypto']
++libraries=['m', 'lzma', 'bz2', 'pthread', 'curl', 'crypto', 'hts']
+ library_dirs=[]
+ print("WITHDEFLATE:", os.getenv('WITHDEFLATE'))
+ if os.getenv('WITHDEFLATE') == "1":
+     print("Using deflate")
+     libraries.append('deflate')
+-    library_dirs.append(os.path.join(dir_path, 'libdeflate'))
++    library_dirs.append(os.path.join(os.environ['EBROOTLIBDEFLATE'], 'lib'))
+ 
+ ffibuilder = FFI()
+ ffibuilder.set_source("libmodbampy",
+@@ -24,11 +24,11 @@
+     """,
+     libraries=libraries,
+     library_dirs=library_dirs,
+-    include_dirs=[src_dir, 'htslib'],
++    include_dirs=[src_dir, os.environ['EBROOTHTSLIB'], 'include'],
+     extra_compile_args=['-std=c99', '-msse3', '-O3'],
+     extra_objects=[
+         'pymod.a',
+-        os.path.join('htslib', 'libhts.a')]
++        os.path.join(os.environ['EBROOTHTSLIB'], 'lib', 'libhts.a')]
```

Comment:
```suggestion
+        ]
```

Removing this means that it'll use the shared library instead.

This will update the patch checksum, so that will need updating in the easyconfig.